module.c

```diff
@@ -1,15 +1,38 @@
-#include <time.h>
-
 #include <Python.h>
+#include <time.h>
 
 #include "third-party/quickjs.h"
 
+// Node of Python callable that the context needs to keep available.
+typedef struct PythonCallableNode PythonCallableNode;
+struct PythonCallableNode {
```

Comment:
`typedef struct PythonCallableNode { ... } PythonCallableNode;`

Reply:
No, that does not work since it refers to itself. Have to have some kind of forward declaration.